src/convert.jl

```diff
@@ -0,0 +1,228 @@
+module Conversion
+
+using ..SAM
+using ..BAM
+
+using BioSequences, BioAlignments
```

Comment:
I recommend you use explicit imports here: Figure out exactly what names you are using an import them.
So, instead of `using BioSequences`, have: `using BioSequences: encode, DNAAlphabet`, etc.
This makes it easier to maintain, since we then know exactly what is needed from the two dependencies.